### ISSUE TITLE
[Bug]: minio s3 sync

### ISSUE THREAD

**Issue by codeses** [sboesen/remotely-sync#79]
### What happened?

When using Minio's S3 synchronization, the synchronization .obsidian configuration directory was set, but after synchronization, there is also a .obsidian directory in Minio's. observer directory
The meaning is that there is still a .obsidian directory under the .obsidian directory
![image](https://github.com/sboesen/remotely-sync/assets/58210554/ae991135-f0b6-4b6b-a208-285005e812cb)

### What OS are you using?

Windows

### What remote cloud services are you using?

S3

### Version of the plugin

0.4.31

### Version of Obsidian

1.4.16

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by sboesen** [comment]
That is super odd, did you figure out why this is happening or a workaround when you closed this issue?

**Comment by codeses** [comment]
It should be a display issue with Minio, similar to the file system returning to the previous level, such as using ../ in Linux
But in mini, it seems to be represented by the name of the higher-level directory. Because I downloaded the synchronized content from the entire mini without any directory structure issues. So I closed the question and answered it on some forums, which also caused me to forget to reply here.
